sdk/resourcemanager/Azure.ResourceManager/tests/Unit/BicepModelReaderWriterOptionsTests.cs

```diff
@@ -0,0 +1,21 @@
+﻿using NUnit.Framework;
+
+namespace Azure.ResourceManager.Tests
+{
+    public class BicepModelReaderWriterOptionsTests
+    {
+        [Test]
```

Comment:
where will the application tests live?  meaning the tests to validate that based on different options the right things change in the serialization?

Reply:
These are tested E2E via the Azure.Provisioning tests.